src/imitation/rewards/reward_nets.py

```diff
@@ -163,78 +185,37 @@ def dtype(self) -> th.dtype:
             return th.get_default_dtype()
 
 
-class ShapedRewardNet(RewardNet):
-    """A RewardNet consisting of a base net and a potential shaping."""
+class RewardNetWrapper(RewardNet, abc.ABC):
```

Comment:
No need to inherit from `abc.ABC` as `RewardNet` already inherits from this. See https://stackoverflow.com/questions/48814211/python3-how-does-one-define-an-abstract-subclass-from-an-existing-abstract-cla

Reply:
Okay. A class only becomes concrete when there are no more `abstractmethod` and `abstractproperty` objects left in the implementation.